### PR TITLE
Changes for ERC7412

### PR DESCRIPTION
**Pull request by noisekit**
Tested on ARB Sepolia. Successfully decoded the new batch `Errors`, collected  24 price ids that require update, added extra price update txn to the multicall and executed without issues

Screenshot 2024-09-23 at 22 50 25

Additionally:
1. refetched missing prices txn after each mutation (as it guarantees price updates are made to succeed)
2. bumped tolerance to 3_300 (55min)
3. deprecated "all prices update" fetcher completely, it was still used for many read txns. Now we only attach missing prices
4. fixed a few random query errors from logs

### CODE REVIEW

File: liquidity/lib/useBorrow/useBorrow.tsx [hunk 1]
```diff
@@ -88,6 +88,10 @@ export const useBorrow = ({
         throw error;
       }
     },
+    onSuccess: () => {
+      // After mutation withERC7412, we guaranteed to have updated all the prices, dont care about await
+      refetchPriceUpdateTx();
```
Comment (dbeal-eth) [inline liquidity/lib/useBorrow/useBorrow.tsx:93]: ooc why do we call this here if we already have all the price updates?

File: liquidity/lib/useLiquidityPositions/useLiquidityPositions.ts [hunk 2]
```diff
@@ -57,14 +55,13 @@ export const useLiquidityPositions = ({ accountId }: { accountId?: string }) =>
...
       },
     ],
     staleTime: 60000 * 5,
-    enabled: !!accountId,
+    enabled: Boolean(network && provider && CoreProxy && accountId && collateralTypes && pools),
     queryFn: async () => {
-      if (!pools || !collateralTypes || !CoreProxy || !accountId || !network || !provider) {
-        throw Error('Query should not be enabled');
+      if (!(network && provider && CoreProxy && accountId && collateralTypes && pools)) {
+        throw Error('OMG');
```
Comment (dbeal-eth) [inline liquidity/lib/useLiquidityPositions/useLiquidityPositions.ts:64]: what is this error?

File: liquidity/lib/useLiquidityPositions/useLiquidityPositions.ts [hunk 3]
```diff
@@ -57,14 +55,13 @@ export const useLiquidityPositions = ({ accountId }: { accountId?: string }) =>
         pools: pools ? pools.map((pool) => pool.id).sort() : [],
         tokens: collateralTypes ? collateralTypes.map((x) => x.tokenAddress).sort() : [],
         priceUpdateTx: stringToHash(priceUpdateTx?.data),
-        CoreProxy: !!CoreProxy,
```
Comment (dbeal-eth) [inline liquidity/lib/useLiquidityPositions/useLiquidityPositions.ts:60]: shouldn't the query be disabled entirely if !CoreProxy?

File: liquidity/lib/useBorrow/useBorrow.tsx (hunk 1 above)
Comment (noisekit) [reply liquidity/lib/useBorrow/useBorrow.tsx:93]: we update it to have empty array

File: liquidity/lib/useLiquidityPositions/useLiquidityPositions.ts (hunk 2 above)
Comment (noisekit) [reply liquidity/lib/useLiquidityPositions/useLiquidityPositions.ts:64]: it should never happen as the query should not be enabled if this condition is true.

File: liquidity/lib/useLiquidityPositions/useLiquidityPositions.ts (hunk 3 above)
Comment (noisekit) [reply liquidity/lib/useLiquidityPositions/useLiquidityPositions.ts:60]: yes but we still need query key, and this is not a correct thing to put in query key.

**Pull request merged**